components/CollectiveContactForm.js

```diff
@@ -1,124 +0,0 @@
-import React from 'react';
```

Comment:
In terms of architecture, we're not gaining much in grouping this component with `ContactCollectiveModal`. Smaller components = more reusability. Having something like:

```jsx
const ContactCollectiveModal = () => (
  <Modal>
    <CollectiveContactForm />
  </Modal>
);
```

- Means that we'll be able to re-use `CollectiveContactForm` elsewhere (edit: we actually need that to not break the contact page)
- Is better for testing. Think about https://styleguide.opencollective.com, testing and iterating on the component is easier if you don't need to open an additional modal

Reply:
Ah, good point, I've used this approach. 👍🏽